sklearn/utils/tests/test_pprint.py

```diff
@@ -174,7 +174,9 @@ def __init__(self, missing_values=np.nan, strategy="mean",
         self.copy = copy
 
 
-def test_basic():
+import pytest
+@pytest.mark.parametrize('value', [False])
+def test_basic(print_changed_only_setter):
```

Comment:
hah, I wasn't aware this was possible. A bit magical but OK.

Reply:
I did not know either... A bit too magical (and verbose to my taste). See: https://github.com/scikit-learn/scikit-learn/pull/17061#discussion_r415992236